### ISSUE TITLE
Make setting boxes larger?

### ISSUE THREAD

**Issue by thegodfather001** [PyroAVR/widget-super-touchplate#3]
Is it possible to make the settings boxs larger? I cant see whats already in the boxes and what i'm typing.

Thank you,

Cameron

**Comment by PyroAVR** [comment]
It may be possible, but within the constraints of the widget this may be difficult to do without causing collisions. Currently I am unable to access the TinyG workspace to see if anything has changed, I'm afraid I've been away for quite a while.  Would it be an acceptable solution to allow the widget to be "maximized" into a sort of modal popup that gives more space to layout components so that each component is more easily viewable?

**Comment by thegodfather001** [comment]
Yeah thats what I was kinda thinking why it was the way it was. In the tutorial video it seemed easier to see. Something must have changed since then. No problem I know how that can be! Yes a “maximized” popup would be prefect. Or I was even thinking of doing away with the inch/mm sign next to each input box cause the units of measurement are already selected and make the boxes bigger. I don’t even know where to begin on doing something like that myself, so I definitely appreciate your help! What’s every easiest.

Thanks again,

Cameron

On Jul 18, 2018, at 8:54 PM, Andy <notifications@github.com<mailto:notifications@github.com>> wrote:


It may be possible, but within the constraints of the widget this may be difficult to do without causing collisions. Currently I am unable to access the TinyG workspace to see if anything has changed, I'm afraid I've been away for quite a while. Would it be an acceptable solution to allow the widget to be "maximized" into a sort of modal popup that gives more space to layout components so that each component is more easily viewable?

**Comment by PyroAVR** [comment]
That was something I struggled with myself, I was developing it on a pretty low-resolution  screen and I did my best to fit everything, but professionally I am an embedded systems engineer, so web development is quite a large jump from what I'm used to.  I believe it should be a simple matter to make a new layout which can handle being in a bootstrap modal, I'll have to get back into that space and see how that is done.
